services/web/server/tests/unit/with_dbs/03/test_users.py

```diff
@@ -789,3 +791,56 @@ async def test_list_permissions_with_no_group_defined_returns_default_false_for_
 
     assert override_services_specifications.name == "override_services_specifications"
     assert override_services_specifications.allowed is False
+
+
+@pytest.mark.parametrize("user_role", [UserRole.USER])
+async def test_mark_account_for_deletion(
+    client: TestClient, logged_user: UserInfoDict, mocker: MockerFixture
+):
+    mock = mocker.patch(
+        "simcore_service_webserver.email._core._do_send_mail",
+        spec=True,
+    )
+
+    # is logged in
+    response = await client.get("/v0/me")
+    await assert_status(response, web.HTTPOk)
```

Comment:
I would also try to delete the user before login to check that it cannot be deleted if the user is not logged in.

Also if I'm user1 and try to delete user2 (and I somehow know his password) this should not be allowed.